tap_partnerize/client.py

```diff
@@ -70,19 +137,12 @@ def get_next_page_token(
         Returns:
             The next pagination token.
         """
-        # TODO: If pagination is required, return a token which can be used to get the
-        #       next page. If this is the final page, return "None" to end the
-        #       pagination loop.
-        if self.next_page_token_jsonpath:
-            all_matches = extract_jsonpath(
-                self.next_page_token_jsonpath, response.json()
-            )
-            first_match = next(iter(all_matches), None)
-            next_page_token = first_match
-        else:
-            next_page_token = response.headers.get("X-Next-Page", None)
-
-        return next_page_token
+        current_token = datetime.datetime.strptime(self.next_page_token, "%Y-%m-%d")
```

Comment:
really common convention in python - since people are almost always using something within `datetime.datetime`, you could do `from datetime import datetime` and then these function calls just become `datetime.strptime(...)` instead

Reply:
Ahh yeah I've seen that around. I'll clean that up, good call.

Reply:
Done, much cleaner. 🧹